src/Utils.cpp

```diff
@@ -41,6 +41,14 @@ BigInt< 4 > getSelectorFromName( const char* name, size_t len )
     return numMask & reversed;
 }
 
+BigInt< 4 > strToBigInt( const char* str)
```

Comment:
I would suggest renaming to `strToBigInt256` since `str` can have value larger than `BigInt<4>` but `strToUint64ArrayImpl` will just ignore it.

Reply:
removed this function as it duplicates `strToBigInt()` from `Utils.hpp`